src/views/v2/components/read-more.php

```diff
@@ -9,15 +9,16 @@
  *
  * @link {INSERT_ARTCILE_LINK_HERE}
  *
- * @version 4.9.10
+ * @version TBD
  *
+ * @var string $permalink The permalink.
  */
+
 ?>
-<div class="tribe-events-c-small-cta tribe-common-b3 tribe-events-c-small-cta--readmore">
+<span class="tribe-events-c-read-more-hellip"> &hellip; </span>
+<div class="tribe-events-c-small-cta tribe-common-b3 tribe-events-c-read-more">
 	<a
-		href="<?php echo esc_url( get_permalink( get_the_ID() ) ) ?>"
+		href="<?php echo esc_url( $permalink ); ?>"
```

Comment:
```suggestion
		href="<?php echo esc_url( $event->permalink ); ?>"
```

Reply:
@bordoni we already discussed this here https://github.com/moderntribe/the-events-calendar/pull/2741#discussion_r333855348 and it's not possible to do that.